pulp_2_tests/constants.py

```diff
@@ -643,3 +643,7 @@
 
 Built from :data:`SRPM_UNSIGNED_FEED_URL` and :data:`SRPM`.
 """
+
+RPM_YUM_METADATA_FILE = 'https://repos.fedorapeople.org/' \
```

Comment:
We have been avoid this kind of break line in code basis.  Just add a flag to make the linter happy.

Reply:
What about add a note to inform this repo was not generated by pulp-fixtures?
```rst
.. NOTE:: This repository is not generated by `pulp-fixtures`_.
.. _pulp-fixtures: https://repos.fedorapeople.org/pulp/pulp/fixtures/
```